zipkin-lens/testdata/yelp.json

```diff
@@ -0,0 +1,360 @@
+    "id": "f5f268651b2a2b34",
+    "kind": "SERVER",
+    "name": "post /location/update/v4",
+    "timestamp": 1571896375297103,
+    "duration": 41740,
+    "localEndpoint": {
+      "serviceName": "mobile_api",
+      "port": 31049
+    },
+    "tags": {
+      "ecosystem": "prod",
+      "habitat": "uswest1bprod",
+      "http.route": "/location/update/v4",
+      "http.uri": "/location/update/v4",
+      "http.uri.qs": "/location/update/v4",
+      "region": "uswest1-prod",
+      "response_status_code": "200",
+      "version_SHA": "6535284b1699df0a766384a648dc95c462a7313d"
+    },
+    "shared": true
+  },
+  {
+    "traceId": "a03ee8fff1dcd9b9",
+    "parentId": "2e8cfb154b59a41f",
+    "id": "f5f268651b2a2b34",
+    "kind": "CLIENT",
+    "name": "post",
+    "timestamp": 1571896375287000,
+    "duration": 56000,
+    "localEndpoint": {
+      "serviceName": "yelp-main"
+    },
+    "remoteEndpoint": {
+      "serviceName": "mobile_api",
+      "port": 31049
+    },
+    "tags": {
+      "client_status_code": "200",
+      "http.uri.client": "<uri_path>",
+      "request_budget": "10003",
+      "tracer": "syslog2scribe.haproxy-synapse"
+    }
+  },
+  {
+    "traceId": "a03ee8fff1dcd9b9",
+    "parentId": "241cea1aa4cb2884",
+    "id": "0facde7c9130fd93",
+    "kind": "CLIENT",
+    "name": "get_multi ***********",
+    "timestamp": 1571896375272125,
+    "duration": 233,
+    "localEndpoint": {
+      "serviceName": "yelp-main",
+      "port": 31523
+    },
+    "remoteEndpoint": {
+      "serviceName": "memcache"
+    },
+    "tags": {
+      "driver": "core_memcache",
+      "hits": "1",
+      "method": "get_multi",
+      "requests": "1"
+    }
+  },
+  {
+    "traceId": "a03ee8fff1dcd9b9",
+    "parentId": "241cea1aa4cb2884",
+    "id": "50b57281525a99d8",
+    "kind": "CLIENT",
+    "name": "commit",
+    "timestamp": 1571896375272604,
+    "duration": 374,
+    "localEndpoint": {
+      "serviceName": "yelp-main",
+      "port": 31523
+    },
+    "remoteEndpoint": {
+      "serviceName": "mysql"
+    },
+    "tags": {
+      "query": "COMMIT"
+    }
+  },
+  {
+    "traceId": "a03ee8fff1dcd9b9",
+    "parentId": "241cea1aa4cb2884",
+    "id": "2b68987704862c4f",
+    "kind": "CLIENT",
+    "name": "get user_details_cache-20150901",
+    "timestamp": 1571896375270438,
+    "duration": 1068,
+    "localEndpoint": {
+      "serviceName": "yelp-main",
+      "port": 31523
+    },
+    "remoteEndpoint": {
+      "serviceName": "memcache"
+    },
+    "tags": {
+      "driver": "core_memcache",
+      "hits": "1",
+      "method": "get",
+      "requests": "1"
+    }
+  },
+  {
+    "traceId": "a03ee8fff1dcd9b9",
+    "parentId": "668ed78ad94b35a1",
+    "id": "241cea1aa4cb2884",
+    "name": "txn: user_get_basic_and_scout_info",
+    "timestamp": 1571896375269210,
+    "duration": 3884,
+    "localEndpoint": {
+      "serviceName": "yelp-main",
+      "port": 31523
+    },
+    "tags": {
+      "calling_method": "************"
```

Comment:
similar, can you replace with a representative value? This could help tell if this is a line of code or something else. The actual line of code and file name, if that, doesn't matter.